weave-js/src/components/Checkbox/Checkbox.tsx

```diff
@@ -59,14 +59,14 @@ export const Checkbox = ({
             'relative rounded-[1.5px]',
             'radix-state-checked:bg-teal-500 radix-state-indeterminate:bg-transparent radix-state-unchecked:bg-transparent',
             {
-              'h-[7.5px] w-[7.5px]': isSmall,
+              'h-8 w-8': isSmall,
```

Comment:
updating checkbox inner size otherwise the ui has to figure out the half pixel and makes it look wonky

https://github.com/wandb/weave/assets/25037002/3c314465-cb34-4cb8-aa3f-caa39a363791